spec/parentheses_matcher_spec.rb

```diff
@@ -1,4 +1,4 @@
-require 'parenthesis_matcher'
+require 'parentheses_matcher'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.